stable/kube-state-metrics/README.md

```diff
@@ -0,0 +1,23 @@
+# kube-state-metrics Helm Chart
+
+* Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install --name my-release incubator/kube-state-metrics
+```
```

Comment:
Install instructions should just be: `helm install stable/kube-state-metrics`—no need to add a repo.